src/vue/backend/src/templates/GeneralSettings.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+    <idx-block className="general-settings">
+        <idx-form-group>
+            <idx-form-label for="website-wrapper"><strong>Name Your Global Website Wrapper</strong><br> Wrappers set the overall styling of your IDX Broker pages, some words about maintaining a consistent design between WordPress and IDX Broker.</idx-form-label>
+            <idx-form-input
+                type="text"
+                id="website-wrapper"
+                :value="wrapperName"
+                @input="generalSettingsStateChange({ key: 'wrapperName', value: $event })"
+            />
+        </idx-form-group>
+        <idx-form-group>
+            <idx-block className="control-label"><strong>Google reCAPTCHA</strong><br> Google reCAPTCHA v3 helps to prevent spammers from filling out your forms.</idx-block>
+            <idx-block className="control-toggle-slider">
+                {{ toggleLabel }}
+                <idx-toggle-slider
+                    uncheckedState="No"
+                    checkedState="Yes"
+                    @toggle="generalSettingsStateChange({ key: 'reCAPTCHA', value: !reCAPTCHA })"
+                    :active="reCAPTCHA"
+                    :label="toggleLabel"
+                ></idx-toggle-slider>
+            </idx-block>
+        </idx-form-group>
+        <idx-form-group>
+            <idx-form-label for="update-frequency"><strong>Update Frequency</strong><br> Choose how often IMPress gets updates from your IDX Broker account.</idx-form-label>
+            <idx-custom-select
+                id="update-frequency"
+                ariaLabel="Select frequency"
+                :selected="updateFrequency"
+                :options="updateFrequencyOptions"
+                @selected-item="generalSettingsStateChange({ key: 'updateFrequency', value: $event })"
+            />
+        </idx-form-group>
+    </idx-block>
+</template>
+<script>
+import { mapState, mapActions } from 'vuex'
+export default {
+    name: 'GeneralSettings',
+    data () {
+        return {
+            toggleLabel: 'Enable Google reCAPTCHA',
+            updateFrequencyOptions: [
```

Comment:
Here is the list of frequency options. I will update the ticket
<img width="149" alt="Screen Shot 2021-02-18 at 2 11 07 PM" src="https://user-images.githubusercontent.com/25048313/108428439-6d51c880-71f3-11eb-9d3d-241844356bd9.png">